converter/src/main/java/gov/cms/qpp/conversion/model/error/Detail.java

```diff
@@ -77,17 +76,35 @@ public static Detail forErrorCode(LocalizedError error) {
 		return detail;
 	}
 
+	private static String computeLocation(Node node) {
+
+		String location = null;
+
+		Node importantParentNode = node.findParentNodeWithHumanReadableTemplateId();
+
+		if (importantParentNode != null) {
+			String importantParentTitle = importantParentNode.getType().getHumanReadableTitle();
+			String possibleMeasureId = importantParentNode.getValue("measureId");
+
+			location = importantParentTitle;
+
+			if (!StringUtils.isEmpty(possibleMeasureId)) {
+				location += " " + possibleMeasureId;
```

Comment:
do we want any extra separation here?

Reply:
What do you mean by separation?  As in extra lines between the code?